ultraLatency.py

```diff
@@ -53,6 +54,11 @@
     print("Invalid MSPL specified: %d, must be in range [1,8]"%(options.MSPL))
     exit(1)
 
+if options.stepSize:
+    step = options.stepSize
+    if (step + options.scanmin > options.scanmax):
+        step = options.scanmax - options.scanmin
+
 if options.debug:
     uhal.setLogLevelTo(uhal.LogLevel.INFO)
```

Comment:
This case structure at line 57 is not needed.  Remove, only the case structure at line 59-60 should be used.   

Additionally you need to assign the value of step before the case structure, otherwise you will have a scope issue, e.g. at line 193 `step=0` because of this scope issue.